backend/app/graphql/meal_request.py

```diff
@@ -41,7 +41,7 @@ class CreateMealRequestResponse(graphene.ObjectType):
 class DonationInfo(graphene.ObjectType):
     donor = graphene.Field(User)
     commitment_date = graphene.DateTime()
-    meal_description = graphene.String()
+    food_description = graphene.String()
```

Comment:
Lets keep it as `meal_description` so that we keep the standard on the backend to always call everything "meal" instead of "food" to avoid confusion.

I apologize for writing food description in the ticket, that was my mistake, sorry! 

You should just be able to do do a search and replace in the backend folder so it shouldn't be to bad to revert!

Reply:
Done!